### DISCUSSION TITLE
Container Distribution Framework

### DISCUSSION THREAD

**Discussion by nadako** [alandtse/skyrim_vr_address_library#74] [Ideas]
Fairly new framework for distributing items into containers. Used by the latest version of the awesome [C.O.I.N.](https://www.nexusmods.com/skyrimspecialedition/mods/51439) mod.

Nexus: https://www.nexusmods.com/skyrimspecialedition/mods/120152
GitHub: https://github.com/SeaSparrowOG/DynamicContainerInventoryFramework
Fork for 1.5: https://github.com/Fuzzlesz/DynamicContainerInventoryFramework-1.5

**Comment by nadako** [comment]
Another one by the same author(s), often used together is Currency Swapper.

Nexus: https://www.nexusmods.com/skyrimspecialedition/mods/127686
GitHub: https://github.com/SeaSparrowOG/CurrencySwapper
Fork for 1.5: https://github.com/Fuzzlesz/CurrencySwapper-1.5

**Comment by treaka** [comment]
Seconded for these two - lots of really cool work by Tate Taylor on making regional currencies. 

Great economy control for excess gold.

**Reply by andperks6** [reply]
I just got the addresses merged to address lib vr and have a VR branch up [here](https://github.com/andperks6/DynamicContainerInventoryFramework/tree/vr)

Working on getting this up on nexus soon, it would be great to get testing from anyone here

**Reply by andperks6** [reply]
Up on the nexus now
